### DISCUSSION TITLE
FrontEnd Website using API help!

### DISCUSSION THREAD

**Discussion by l-urk** [coinfoundry/miningcore#1566] [General]
Is there some HTML and JavaScript to table file that uses the API example : 

http://localhost:4000/api/pools
http://localhost:4000/api/pools/bitcoin
http://localhost:4000/api/pools/bitcoin/performance
http://localhost:4000/api/pools/bitcoin/miners/<miner-wallet-address

I want to make a custom website frontend for my pool. Is there some fetch JavaScript that I could use for reference to build my own?
I can't seem to find the right information for writing a compatible JSON parsing script to work with these API calls.

Even having them display as a table with no graphics would be good enough. But if I could add some fancy charts and graphs that would be even better.

**Comment by blackmennewstyle** [comment]
You will have to code it yourself. I did it for my mining pool: https://conceal.cedric-crispin.com/

**Comment by xyrox54** [comment]
I use this : https://github.com/minernl/Miningcore.WebUI
